### DISCUSSION TITLE
Question on training multiple faces.

### DISCUSSION THREAD

**Discussion by cgray77** [TheLastBen/fast-stable-diffusion#1166] [Q&A]
The current dreambooth suggests 650 steps for 10 images. If I'm adding more than one subject, should that be increased? IE, if I have two subjects would the number of training steps be increased to 1300?

Also, I can't find this, but the original suggestion was that you should have full body shots among the 30 images. Is this still the case?

Thank you!

**Comment by TheLastBen** [comment]
For 2 subjects, double the number of unet steps, keep the text_encoder below 500

**Comment by meunumerotim333** [comment]
For me 2 persons always mess.. even if the prompt is very unique. So I still prefer to use just one person trained per model.

**Reply by TheLastBen** [reply]
Yes, for people some times the model tends to merge them, that's the downside of the current model's architecture, especially when finetuned on a small dateset.

**Comment by David19910306** [comment]
您的来信已收到，谢谢。。。。
